clients/dashboards/dashboards.go

```diff
@@ -0,0 +1,82 @@
+package dashboards
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/influxdata/influx-cli/v2/api"
+	"github.com/influxdata/influx-cli/v2/clients"
+	"github.com/influxdata/influx-cli/v2/pkg/influxid"
+)
+
+type Client struct {
+	clients.CLI
+	api.DashboardsApi
+	api.OrganizationsApi
+}
+
+type Params struct {
+	clients.OrgParams
+	Ids []string
+}
+
+func (c Client) List(ctx context.Context, params *Params) error {
+	orgID, err := c.getOrgID(ctx, params.OrgID, params.OrgName)
+	if err != nil {
+		return err
+	}
+	if orgID == "" && len(params.Ids) == 0 {
+		return fmt.Errorf("at least one of org, org-id, or id must be provided")
```

Comment:
Should this check run before `getOrgID`? I think if it runs after, `getOrgID` might search for orgs using the empty string as the org name and get back every org.

Reply:
If this gets moved to the front, the condition will need to check `params.OrgName` and `c.ActiveConfig.Org` as well (like in the telegraf commands)

Reply:
This was how it was structured before, but now that you've brought it up, I think this was part of the issue in #108, this and the fact that it didn't check orgName consistently